### ISSUE TITLE
Windows 10 start menu button

### ISSUE THREAD

**Issue by Screemer1** [Open-Shell/Open-Shell-Menu#115]
First of all, thank you for picking this great project up and keeping developing it. I really love it and can't stand the default Win8 - 10 start menu, it sucks. I hope you'll keep going for years. :)

I don't seem to be able to make my chosen start menu button stick in OpenShell v4.4.131
I set the button and it's working fine until I reboot. After reboot a default OpenShell button is shown.
But when I open " Open-Shell Menu Settings " it shows the one I have chosen.
If I reset it to the same file again it shows up as start menu button.

I don't know if this is just my machine, a known bug or reported before. I couldn't find anything about it here though.
When I ran ClassicalStartmenu it worked flawless with this image.

I can't find where I downloaded my image but I uploaded it if you need to check it out. :)
`https://screemer.nu/image-gallery/8-mixedimages/detail/18-win7noorb-png`

Anyone have any idea?

**Comment by CTVCAM8** [comment]
Your start button works ok for me on Open-Shell v4.4.131 after reboot.



![test start button on 4 4 131](https://user-images.githubusercontent.com/44127216/49245783-300d8000-f3c8-11e8-8874-a5f9a091e064.png)



Maybe something left over when upgrading from Classic Start to Open-Shell?
Only suggestion is "Reset all settings" in the Backup button, exit Open-Shell,
and set up custom button again.

**Comment by Screemer1** [comment]
Thank you very much. I had missed out of the setting under advanced settings for start button.
I only set it on the initial screen.
![openshell 2](https://user-images.githubusercontent.com/45452828/49281739-9dcfaf80-f48d-11e8-83a5-0747c54a7969.png)
![openshell 1](https://user-images.githubusercontent.com/45452828/49281740-9e684600-f48d-11e8-9829-cda1c0cf8376.png)
After I had changed the second one to as you suggested it all works. 🥇
